examples/src/main/res/layout/activity_instruction_view_layout.xml

```diff
@@ -50,7 +50,6 @@
 
     <com.mapbox.navigation.ui.instruction.InstructionView
         android:id="@+id/instructionView"
-        style="@style/InstructionView"
```

Comment:
remove the customized style, as the example activity to show how to customize style is under reviewing: https://github.com/mapbox/mapbox-navigation-android/pull/2753